linear-algebra/src/main/scala/scalan/linalgebra/LADsl.scala

```diff
@@ -0,0 +1,16 @@
+package scalan.linalgebra
+
+import scalan.{ScalanDsl, ScalanDslStd, ScalanDslExp}
+
+/**
+ * Created by Victor Smirnov on 12/25/15.
+ */
```

Comment:
Always remove such kind comments. They are useless. The ownership of the code is based on the lines you add or change.